returnn/tf/layers/signal_processing.py

```diff
@@ -464,7 +511,9 @@ def _padTimeSignal(input_placeholder, frame_size):
         concat_feature_dim = channel_wise_stft.shape[2] * channel_wise_stft.shape[3]
         channel_concatenated_stft = tf.reshape(channel_wise_stft, (batch_dim, time_dim, concat_feature_dim))
         if channel_wise_stft_res_list:
-          channel_concatenated_stft = _cropStftOutputToReferenceFrameSizeLength(channel_concatenated_stft, get_shape(channel_wise_stft_res_list[0])[1])
+          channel_concatenated_stft = \
+            _crop_stft_output_to_reference_frame_size_length(channel_concatenated_stft,
+                                                             get_shape(channel_wise_stft_res_list[0])[1])
```

Comment:
Do not use `\`. Instead, add brackets.
Also, do not indent like that. We don't want to have such wide hanging indents.
```suggestion
          channel_concatenated_stft = (
            _crop_stft_output_to_reference_frame_size_length(
              channel_concatenated_stft,
              get_shape(channel_wise_stft_res_list[0])[1]))
```